pkg/backend/display/testdata/not-truncated/template-body.json.interactive-80x24-cooked.stdout.txt

```diff
@@ -221,2303 +221,2303 @@
- <{%reset%}>  <{%reset%}>  │  ├─ awsx:x:ec2:NatGateway       vpc-0                            <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>  │  │  └─ aws:ec2:Eip              vpc-0                           
- <{%reset%}>  <{%reset%}>  │  ├─ awsx:x:ec2:Subnet           vpc-public-0                     <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>  │  │  └─ aws:ec2:RouteTable       vpc-public-0                    
- <{%reset%}>  <{%reset%}>  │  ├─ awsx:x:ec2:Subnet           vpc-private-0                    <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>  │  ├─ awsx:x:ec2:InternetGateway  vpc                              <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>  │  │  └─ aws:ec2:InternetGateway  vpc                             
- <{%reset%}>  <{%reset%}>  │  ├─ awsx:x:ec2:Subnet           vpc-private-1                    <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>  │  │  └─ aws:ec2:RouteTable       vpc-private-1                   
- <{%reset%}>  <{%reset%}>  │  ├─ awsx:x:ec2:Subnet           vpc-public-1                     <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>  │  ├─ awsx:x:ec2:NatGateway       vpc-1                            <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>  │  │  └─ aws:ec2:Eip              vpc-1                           
- <{%reset%}>  <{%reset%}>  │  └─ aws:ec2:Vpc                 vpc                              <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>  └─ eks:index:Cluster              cluster                          <{%reset%}><{%reset%}>   
- <{%reset%}>  <{%reset%}>     ├─ eks:index:ServiceRole       cluster-instanceRole             <{%reset%}><{%reset%}>   
- <{%reset%}>  <{%reset%}>     │  └─ aws:iam:Role             cluster-instanceRole-role        <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>     ├─ eks:index:ServiceRole       cluster-eksRole                  <{%reset%}><{%reset%}>   
- <{%reset%}>  <{%reset%}>     │  └─ aws:iam:Role             cluster-eksRole-role             <{%reset%}><{%reset%}> 
- <{%reset%}>  <{%reset%}>     ├─ eks:index:RandomSuffix      cluster-cfnStackName             <{%reset%}><{%reset%}>   
- <{%bold%}><{%reset%}>  <{%reset%}>     └─ aws:ec2:SecurityGroup       cluster-eksClusterSecurityGroup  <{%bold%}><{%reset%}><{%reset%}>   
+ <{%reset%}>  <{%reset%}>  ├─ awsx:x:ec2:Vpc                 vpc                              <{%reset%}><{%reset%}>       
+ <{%reset%}>  <{%reset%}>  │  ├─ awsx:x:ec2:NatGateway       vpc-0                            <{%reset%}><{%reset%}>       
+ <{%reset%}>  <{%reset%}>  │  │  └─ aws:ec2:Eip              vpc-0                            <{%reset%}><{%reset%}>       
+ <{%reset%}>  <{%reset%}>  │  ├─ awsx:x:ec2:Subnet           vpc-public-0                     <{%reset%}><{%reset%}>       
+ <{%reset%}>  <{%reset%}>  │  │  └─ aws:ec2:RouteTable       vpc-public-0                     <{%reset%}><{%reset%}>       
```

Comment:
Nit: It looks like we can optimize out a colorized column of length 0 and elide it.